### PR TITLE
feat(Stan): fix, refactor, and add cover image

### PR DESCRIPTION
**Pull request by Slowlife01**
**Describe the changes in this pull request:**
Fixes, refactors, and adds cover image support.

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

image
</details>

### CODE REVIEW

File: websites/S/Stan/presence.ts [hunk 1]
```diff
@@ -268,5 +306,10 @@ presence.on("UpdateData", async () => {
       }
     }
   }
-  presence.setActivity(presenceData);
+
+  for (const [key, value] of Object.entries(presenceData))
+    if (value === "undefined") delete presenceData[key as keyof PresenceData];
```
Comment (ImRodry) [inline websites/S/Stan/presence.ts:311]: pretty sure this can never be true and it also seems quite useless

File: websites/S/Stan/presence.ts [hunk 2]
```diff
@@ -50,59 +54,85 @@ const presence = new Presence({
...
-          {
-            label: data.meta.episode ? strings.viewEpisode : "Watch Video",
-            url: document.URL
-          }
-        ];
+          presenceData.buttons = [
+            {
+              label: data.meta.episode
+                ? data.strings.viewEpisode
+                : "Watch Video",
```
Comment (ImRodry) [inline websites/S/Stan/presence.ts:127]: use general.buttonWatchVideo instead

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/S/Stan/presence.ts [hunk 3]
```diff
@@ -50,59 +54,85 @@ const presence = new Presence({
     };
   } = {
     presence: {},
+    coverUrls: {},
+    strings: null,
     meta: {},
     startedSince: ~~(Date.now() / 1000),
```
Comment (ImRodry) [inline websites/S/Stan/presence.ts:60]: I know this PR didn't touch this line but using the bitwise NOT operator yields a slightly misleading value as opposed to Math.round (1 unit smaller)
```suggestion
    startedSince: Math.round(Date.now() / 1000),
```

File: websites/S/Stan/presence.ts [hunk 4]
```diff
@@ -22,6 +23,10 @@ const presence = new Presence({
     meta?: {
       [key: string]: string;
     };
+    strings: Awaited<ReturnType<typeof getStrings>>;
+    coverUrls?: {
+      [key: string]: string;
+    };
     settings?: {
       id?: string;
       delete?: boolean;
```
Comment (ImRodry) [inline websites/S/Stan/presence.ts:32]: Again, up to you if you wanna fix this in this PR but you should probably not use reserved keywords as setting names or properties

**Review by ImRodry — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Pull request merged**